tests/models/identifiers/test_identifiers.py

```diff
@@ -0,0 +1,38 @@
+from lxml import etree
+
+from caselawclient.models.identifiers import Identifier, IdentifierSchema
+
+
+class TestIdentifierSchema(IdentifierSchema):
+    __test__ = False
```

Comment:
what is this?

Reply:
This indicates to pytest that although the class is named as `Test` it doesn't actually contain things which want to be tested (this is the "Test Identifier Schema", not tests _for_ the Identifier Schema).